php/ajax.php

```diff
@@ -56,6 +56,9 @@ function wphc_subscribe_ajax() {
  * Saves the settings from the "Settings" tab of the plugin
  */
 function wphc_save_settings() {
+	if (!current_user_can('manage_options') || !wp_verify_nonce('action', 'action')) {
+        	exit;
+    	}
```

Comment:
@tewen I got this from here: https://patchstack.com/articles/common-plugin-vulnerabilities-how-to-fix-them/#toc-item-4

Also maybe this should be `return` instead of `exit` ??

I don't know if this will actually work or how to check it.

Reply:
I don't see `wp_verify_nonce` used anywhere else in the app so maybe that won't work here.

Reply:
We may need to use `wp_create_nonce` in order to generate the nonce token. https://developer.wordpress.org/reference/functions/wp_create_nonce/